syft/pkg/cataloger/cpe.go

```diff
@@ -98,8 +103,7 @@ func candidateVendors(p pkg.Package) []string {
 
 func candidateProducts(p pkg.Package) []string {
```

Comment:
The python-|_ should be an option in the product name as well

Reply:
righto, adjusted in the [latest commit](https://github.com/anchore/syft/pull/397/commits/924e9e6e6ad268cfe72dd77de250c16e7c68d0a6)